demisto_sdk/commands/unify/unifier.py

```diff
@@ -433,3 +444,41 @@ def add_contributors_support(self, unified_yml: Dict, contributor_type: str, con
         unified_yml['detaileddescription'] = contributor_description + '\n***\n' + existing_detailed_description
 
         return unified_yml
+
+    def get_integration_doc_link(self, unified_yml: Dict) -> str:
+        """Generates the integration link to the integration documentation
+
+        Args:
+            unified_yml (Dict): The integration YAML dictionary object
+
+        Returns:
+            str: The integration doc markdown link to add to the detailed description (if reachable)
+        """
+        normalized_integration_id = self.normalize_integration_id(unified_yml['commonfields']['id'])
+        integration_doc_link = INTEGRATIONS_DOCS_REFERENCE + normalized_integration_id
+
+        try:
+            res = requests.get(integration_doc_link, verify=False, timeout=10)
```

Comment:
Few issues with doing this:
1. it will slow down the creation of the content pack
2. We only publish the doc after it is in master. So we are not running this code while working on the branch (if the integration was not yet merged to master with docs).

I think it will be better to simply check if there is a README.md file in the directory and if so just add the link.